.gitignore

```
@@ -1,4 +1,4 @@
-
 \.idea/
 
 *.pyc
+/experiments/*
```

Shouldn't every experiment store its results in a folder within it? For rainbow I am automatically creating two folders the first time you run an experiment, "results" for plots and tensorboard summaries (2nd I don't use) and "weights" to store trained models.
You can add to ignore the following if we agree on that:
```suggestion
/*/experiments/*
```

---

Within the folder of
/experiments/5_objects_pickup_fp211_easy
I have 
/experiments/5_objects_pickup_fp211_easy/checkpoints
/experiments/5_objects_pickup_fp211_easy/tensorboard_logs
/experiments/5_objects_pickup_fp211_easy/latest_args.json
/experiments/5_objects_pickup_fp211_easy/latest_config.json

So not fully sure what you mean? I can run thousands of experiments, are you saying you can only run 1 with Rainbow?

---

For now yes I always override the old one, but that is completely out of the point here. What I am suggesting is that since algorithms are self-contained messiness they should also contain their own experiments in a folder a3c/experiments, rainbow/experiments and so on. Because the results you are storing might be even totally different as it was the case until now.

---

yeah, good point, i'll do this. Also necessary since PPO could be using visdom and different folder structures. We can copy and paste the experiment folder creation code (might not be DRY but it'll be ok)